src/style.css

```
@@ -1,3 +1,68 @@
+:root {
+  --color-primary: #00acdc; /* Define a variable for primary color */
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
   margin: 0;
```

header doen't have default margin
```suggestion
  display: flex;
```